src/store/ducks/plans.ts

```diff
@@ -76,21 +77,15 @@ export interface Plan extends PlanRecord {
   jurisdiction_path: string[];
 }
 
-/** UseContext - interface for PlanPayload.useContext[] items */
-export interface UseContext {
```

Comment:
Why is this deleted?

Reply:
the reason of removing this interface was to remove the duplication of code we already have this interface defined [here](https://github.com/onaio/reveal-frontend/blob/master/src/configs/settings.ts#L1670) with better type check